src/main/java/RacingCar/Main.java

```diff
@@ -0,0 +1,14 @@
+package RacingCar;
+
+public class Main {
+    public static void main(String[] args) {
+        InputView inputView = new InputView();
+
+        RacingCar racingCar = new RacingCar();
+        racingCar.initCars(inputView.carCount());
+        racingCar.initTryCnt(inputView.tryCount());
+
+        ResultView resultView = new ResultView(racingCar);
+        resultView.printCar();
```

Comment:
이런식으로 변경하면 어떨까요?
생성자를 사용하면 클래스 초기화 기능을 할 수 있고 값에 대한 검증도 같이 수행할 수 있는 장점이 있습니다😄
```suggestion
        InputView inputView = new InputView();

        RacingCar racingCar = new RacingCar(inputView.carCount(), inputView.tryCount());
        RacingCarResult result = racingCar.start();

        ResultView resultView = new ResultView(result);
        resultView.printCar();
```